src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1628499901.inc.php

```diff
@@ -0,0 +1,17 @@
+<h1>Build #1628499901</h1>
+<h2>Date: 2021-08-09</h2>
+<div class="changelog">
+    - #662: add missing cms message type name translations so TCMSLogChange::GetMessageTypeByName works again
+</div>
+<?php
+
+$languageService = \ChameleonSystem\CoreBundle\ServiceLocator::get('chameleon_system_core.language_service');
+$baseLanguage = $languageService->getCmsBaseLanguage();
+
+if ('de' === $baseLanguage->fieldIso6391) {
+    $query = "UPDATE `cms_message_manager_message_type` SET `name__en` = `name` WHERE `name__en` = ''";
```

Comment:
Depending on the system some of these or all are already correctly translated.

So this would make it "wrong" for some entries and on the other hand does not help with loading for the others.

Reply:
I translate only empty english translations, so nothing gets overwritten. The name is used as headlines in error messages btw..